composer.json

```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "php":                       "^7.1.0",
+        "php":                       "^7.2.0",
```

Comment:
You should also change Travis to run builds under 7.2 - it's supported as of today.